com.unity.visualeffectgraph/Documentation~/Reference-Attributes.md

```diff
@@ -23,15 +23,14 @@ Some attributes are a bit more advanced and will be used by default in most simu
 
 | Name       | Type    | Description                                                  | Default Value                                     |
 | ---------- | ------- | ------------------------------------------------------------ | ---------- |
-| `mass` | float | The mass of a particle in Kg/dm^3 | 1.0 (defaults to 1kg per liter of water) |
-| `angle` | Vector3 | **Variadic:** Euler rotation of a simulated element, expressed as a Vector of Degrees Values. | (0,0,0) |
+| `mass` | float | The mass of a particle | 1.0 (defaults to 1kg per liter of water) |
```

Comment:
I would remove "(defaults to 1kg per liter of water)" which does not really make sense